astro/src/content/blog/announcing-fusionauth-1-53.mdx

```diff
@@ -0,0 +1,45 @@
+---
+publish_date: 2024-09-04
+title: Announing FusionAuth Version 1.53.0 - Webhook Logs and New Lambdas
+description: Version 1.53.0 of FusionAuth introduces two hotly-requested features. You can now see a history of webhooks and their statuses, and logins now have more logic.
+authors: Brad McCarty
+image: /img/blogs/release-1-53/fusionauth-1-53-0.png
+categories: Product
+tags: lambdas, login logic, webhook logs, customer data 
+excerpt_separator: "{/* more */}"
+---
+
+We recently released version 1.53.0 of FusionAuth. As we [continue our work](/blog/fusionauth-inc-5000-2024) toward simplification and a better user experience, we've added some hotly-requested features. Marching forward with our animal-based naming convention, I hereby dub version 1.53.0 The Login Lambda Llama.
+
+<center>
+![FusionAuth Login Lambda Llama Meme. A llama with the caption that reads Lambda Llama is suspicious of your credentials.](/img/blogs/release-1-53/llama.jpg "login lambda llama")
+</center>
+## Lambda-Based Login Control
+
+Sometimes you need fine-grained control of an authentication. Here are some examples:
+
+- A person who is a member of one group, but not of another. Weekday access to an application depends on being in both groups.
+- You want to require all users who are logging in to use SAMLv2 and disallow any other kinds of login.
+- Users of the Hooli application must have a hooli.com email address, while users of the Pied Piper application can have a hooli.com or a piedpiper.com email address.
+
+In a scenario like these, you need more than a login failure. After all, their login information may be correct, but they just don't have access to that application. You need to be able to apply some level of logic to their authentication event. That's where login lambdas come into play. You can find a high-level overview of lambdas [inside our docs](/docs/extend/code/lambdas/).
+
+This is an addition to our long list of lambdas, which all add flexibility and extensibility to the FusionAuth login experience.
+
+In this release, there is a new lambda that runs at the moment of login. This lambda returns a list of themeable error messages. If the list is empty, the login process continues. If the list contains error messages, then the login fails and your users are presented with the error messages.
+
+This kind of flexibility should help developers use any kind of business logic to prevent unwanted access to their applications. When combined with Lambda HTTP Connect, you can even call out to external APIs and re-use existing business logic to allow or deny access.
```

Comment:
```suggestion
This kind of flexibility should help developers use any kind of business logic to prevent unwanted access to their applications. When combined with [Lambda HTTP Connect](/docs/extend/code/lambdas/#using-lambda-http-connect), you can even call out to external APIs and re-use existing business logic to allow or deny access.
```